genfragments/ThirteenPointSixTeV/ZZ/ZZToAny_TuneCP5_13p6TeV_pythia8_cfi.py

```diff
@@ -0,0 +1,37 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+from Configuration.Generator.PSweightsPythia.PythiaPSweightsSettings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
```

Comment:
generator = cms.EDFilter("Pythia8ConcurrentGeneratorFilter",

Reply:
Hi, do you know why BTV did this sample? I was requested by Higgs Gen group to produce these samples. It is fine if they did it, I just want to know if the sample I was requested to make is the same as this one.